packages/analyzer/fixtures/02-inheritance/05-external-in-monorepo/monorepo/packages/to-be-analyzed-pkg/package.json

```diff
@@ -4,11 +4,12 @@
   "type": "module",
   "dependencies": {
     "@mono/sibling-pkg": "0.2.0",
-    "ext-pkg-without-export-map": "0.1.0"
+    "ext-pkg-without-export-map": "0.1.0",
+    "is-sorted": "1.0.5"
```

Comment:
I added this dependency to ensure that the test fails if the regex is not present, it passes otherwise.